llvm/unittests/CodeGen/ScheduleDAGMITestUtils.cpp

```diff
@@ -26,12 +26,19 @@ MCInstrDesc MCIDs[] = {
     {TargetOpcode::DBG_VALUE, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0},
     {TargetOpcode::GENERIC_OP_END + 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0}};
 
+std::unique_ptr<MachineSchedStrategy> getSchedStrategy(MachineSchedContext *C,
```

Comment:
Oopsie: Those changes to ScheduleDAGMITestUtils were meant to be squashed in the previous commit, not this one.